packages/flutter/lib/src/widgets/icon_data.dart

```diff
@@ -66,3 +66,43 @@ class IconData {
   @override
   String toString() => 'IconData(U+${codePoint.toRadixString(16).toUpperCase().padLeft(5, '0')})';
 }
+
+/// [DiagnosticsProperty] that has an [IconData] as value.
+class IconDataDiagnosticsProperty extends DiagnosticsProperty<IconData> {
```

Comment:
@pq This one also needs a lint so we don't get bugs when someone uses DiagnosticsProperty<IconData> instead.